internal/plugincommon/discovery/processor_test.go

```diff
@@ -41,20 +43,31 @@ func TestContractDiscoveryProcessor_Observation_SupportsDest_HappyPath(t *testin
 	expectedOnRamp := map[cciptypes.ChainSelector][]byte{
 		source: []byte("onRamp"),
 	}
+	expectedFeeQuoter := map[cciptypes.ChainSelector][]byte{
+		source: []byte("onRamp"),
```

Comment:
Why are we adding source as onRamp here?

Reply:
This is the fee quoter from the onramp, the other entry here is a fee quoter from the destination. I updated the names for clarity.